spec/spec_helper.rb

```diff
@@ -5,6 +5,9 @@
 require 'app/application'
 require 'rspec'
 require 'rack/test'
+require 'coveralls'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.